server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -105,7 +109,7 @@ export default class BulkChangeModal extends React.PureComponent {
     this.mounted = false;
   }
 
-  handleCloseClick = (e: Event & { target: HTMLElement }) => {
+  handleCloseClick = (e /*: Event & { target: HTMLElement } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)